tests/system/Database/Live/GetTest.php

```diff
@@ -229,7 +229,15 @@ public function testGetRowWithReturnType()
 
     public function testGetRowWithCustomReturnType()
     {
-        $testClass = new class () {};
+        $testClass = new class () {
```

Comment:
How about using `stdClass` to skip adding these properties?

Reply:
The test is for Custom Return Type, not `'object'`.